### PR TITLE
Add SIG Docs Arabic team meeting to README

### PR DESCRIPTION
**Pull request by adowair**
This commit will add details of the Arabic localization team's regular meeting to the SIG Docs README. This meeting was setup at the request of the team. Find the starting conversation here: https://kubernetes.slack.com/archives/C0191RDKHU1/p1706210213318239

**Which issue(s) this PR fixes**:
None

### CODE REVIEW

**Comment by cblecker** [comment]
This change needs to be made in sigs.yaml and `make` run

File: sig-docs/README.md [hunk 1]
```diff
@@ -17,6 +17,9 @@ The [charter](charter.md) defines the scope and governance of the Docs Special I
 * APAC SIG Meeting: [Wednesdays at 05:30 UTC](https://docs.google.com/document/d/1emuO4nmaQq3K8JZ9-MQeIygtrCPO9kWv7U7RzTaW4F8/edit) (monthly - Wednesday, after the fourth Tuesday, every month). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=05:30&tz=UTC).
   * [Meeting notes and Agenda](https://docs.google.com/document/d/1emuO4nmaQq3K8JZ9-MQeIygtrCPO9kWv7U7RzTaW4F8/edit).
   * [Meeting recordings](https://www.youtube.com/playlist?list=PL69nYSiGNLP3b5hlx0YV7Lo7DtckM84y8).
+* Arabic Team Meeting: [Wednesdays at 18:00 UTC](https://zoom.us/j/98193594774?pwd=VzlweTE3ckxscDRhWVFtRHp2Znlxdz09) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=18:00&tz=UTC).
+  * [Meeting notes and Agenda](hhttps://docs.google.com/document/d/1s4KdaDEYGR-hGkxIl2sRwyXZpuVgeQD6XNxcISkA-vE).
```
Comment (a-mccarthy) [inline sig-docs/README.md:21]: looks like there is an extra `h` in the front of the URL

File: sigs.yaml [hunk 2]
```diff
@@ -1576,6 +1576,14 @@ sigs:
     url: https://docs.google.com/document/d/1emuO4nmaQq3K8JZ9-MQeIygtrCPO9kWv7U7RzTaW4F8/edit
     archive_url: https://docs.google.com/document/d/1emuO4nmaQq3K8JZ9-MQeIygtrCPO9kWv7U7RzTaW4F8/edit
     recordings_url: https://www.youtube.com/playlist?list=PL69nYSiGNLP3b5hlx0YV7Lo7DtckM84y8
+  - description: Arabic Team Meeting
+    day: Wednesday
+    time: "18:00"
+    tz: UTC
+    frequency: biweekly
+    url: https://zoom.us/j/98193594774?pwd=VzlweTE3ckxscDRhWVFtRHp2Znlxdz09
+    archive_url: hhttps://docs.google.com/document/d/1s4KdaDEYGR-hGkxIl2sRwyXZpuVgeQD6XNxcISkA-vE
```
[the remark this answers is not in the document]
Comment (adowair) [reply sigs.yaml]: Good catch, thanks @a-mccarthy. Fixed here: https://github.com/kubernetes/community/pull/7752/commits/2811076933c4644eebb1e8d5d6b75613f2886a83

**Comment by adowair** [comment]
Hi @a-mccarthy, is there anything else we need to review+merge this PR? Thank you bunches

File: sig-docs/README.md [hunk 3]
```diff
@@ -17,6 +17,9 @@ The [charter](charter.md) defines the scope and governance of the Docs Special I
 * APAC SIG Meeting: [Wednesdays at 05:30 UTC](https://docs.google.com/document/d/1emuO4nmaQq3K8JZ9-MQeIygtrCPO9kWv7U7RzTaW4F8/edit) (monthly - Wednesday, after the fourth Tuesday, every month). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=05:30&tz=UTC).
   * [Meeting notes and Agenda](https://docs.google.com/document/d/1emuO4nmaQq3K8JZ9-MQeIygtrCPO9kWv7U7RzTaW4F8/edit).
   * [Meeting recordings](https://www.youtube.com/playlist?list=PL69nYSiGNLP3b5hlx0YV7Lo7DtckM84y8).
+* Arabic Team Meeting: [Wednesdays at 17:00 UTC](https://zoom.us/j/98193594774?pwd=VzlweTE3ckxscDRhWVFtRHp2Znlxdz09) (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=17:00&tz=UTC).
```
Comment (divya-mohan0209) [inline sig-docs/README.md:20]: The timezone converter shows 5 PM UTC, while the agenda doc shows 6 PM UTC as the time for the meeting. Please amend either of the two & let us know for further review.

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]

File: sig-docs/README.md (hunk 3 above)
Comment (adowair) [reply sig-docs/README.md:20]: Thanks that catch @divya-mohan0209, I've gone ahead and updated the agenda doc to reflect the correct time **17:00 UTC**.

**Comment by adowair** [comment]
Bumping this PR 😄, thank you for the reviews

**Review by divya-mohan0209 — COMMENTED** [review]
This PR needs a rebase, @adowair . Once the changes are done, happy to lgtm

**Comment by adowair** [comment]
> This PR needs a rebase, @adowair . Once the changes are done, happy to lgtm


Thank you @divya-mohan0209, I've gone ahead and rebased

**Comment by adowair** [comment]
This PR is superseded by #8003.

**Pull request closed without merging**